src/test/shared/regions/regionUtilities.test.ts

```diff
@@ -66,7 +66,7 @@ describe('getRegionsForActiveCredentials', async function () {
     })
 
     it('defaults to the standard partition if no default region is found', async function () {
-        fnGetCredentialDefaultRegion.returns(undefined)
+        fnGetCredentialDefaultRegion.returns('us-east-1')
 
         getRegionsForActiveCredentials(awsContext, regionProvider)
         assert.ok(fnGetPartitionId.alwaysCalledWith('us-east-1'), 'expected default region to be used')
```

Comment:
Need to think about what this test was checking. Did we trick the test into passing? Or is it correct that getCredentialsDefaultRegion() no longer returns undefined?

Reply:
it was set to return undefined in order that it would fall to use DEFAULT_REGION, which in other words is the same as setting it to 'us-east-1' (DEFAULT_REGION)

Reply:
The type definition has changed, so in this case it does make sense to change the test. The stub does not have types set-up correctly, otherwise returning `underfined` shows as a TS error.